pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,491 @@
+within this container. This ensures a consistent and isolated setting for all
+your database tasks. From a single image, you can spawn multiple
+containers. Each container functions as a distinct Memgraph instance.
+</Callout>
+
+
+Images are downloaded using the `docker pull` command followed by the name of
+the Docker image. We encourage you to use Memgraph Platform as it includes
+everything you might need while making the best of Memgraph.
+
+To download the latest Memgraph Platform image, run:
+
+```
+docker pull memgraph/memgraph-platform
+```
+
+All Memgraph Docker images are availble at [Memgraph's Docker
+Hub](https://hub.docker.com/u/memgraph). There you will find exact Docker image
+tags you can downalod and use to install Memgraph with. 
+
+### Architecture of Docker container running Memgraph
+
+The picture below shows the architecture of the Memgraph Docker ecosystem.
+
+![](/pages/getting-started/install-memgraph/docker/docker-architecture.png)
+
+### Run a Memgraph Docker image
+
+All images are started using the `docker run` command followed by various flags,
+environment variables and configuration options.
+
+The most common flags used while running Memgraph images are:
+
+- publish ports: `-p 3000:3000`
+- container name: `--name memgraph`
+- set up configuration using environment variables in the case of the
+  `memgraph-platform` image, or configuration flags using the `memgraph` or
+  `memgraph-mage` image
+
+A `docker run` command can look like this:
+
+```
+docker run -p 7687:7687 [-p host_port:container_port] [--name container_name] [configuration] memgraph/image_name
+```
+
+#### Publish ports
+
+Ports are published in order to allow services outside the container to be able
+to connect to the container. When publishing ports, you need to define two ports
+separated by a colon. The left side port stands for the **host port** and the
+right side port is the **container port**.
+
+The most common ports published while running Memgraph are:
+
+- `-p 7687:7687` - connection to the database instance (the Bolt protocol uses
+  this port by default)
+- `-p 3000:3000` - connection to the Memgraph Lab application when running
+  Memgraph Platform
+- `-p 7444:7444` - connection to fetch log files from Memgraph Lab (only in
+  version 2.0 and newer)
+
+For example, if you are running two instances using the `memgraph-platform`
+image and you want to connect to both instances using the Memgraph Lab
+in-browser application. You would run the first instance with:
+
+```
+docker run -p 7444:7444 -p 3000:3000 memgraph/memgraph-platform
+```
+
+Because port `3000` is now taken, you need to change the left side
+port (host ports):
+
+```
+docker run -p 7444:7444 -p 3001:3000 memgraph/memgraph-platform
+```
+
+To connect to the first instance, you should open Memgraph Lab in your browser
+at `localhost:3000`, but the second instance is reachable at `localhost:3001`.
```

Comment:
maybe we should also explain that this change in 7444 port is for external services